test/unit/question_base_test.rb

```diff
@@ -6,11 +6,11 @@ class QuestionBaseTest < ActiveSupport::TestCase
   end
 
   context '#next_node' do
-    should 'ensure single next node key is supplied if next_node called without block' do
+    should 'raise exception if next_node is called without a block' do
```

Comment:
Argh, I think I totally misread the test name! I guess I was more questioning if we calling next_node, or defining a next_node in the flow.